js/chatbot/src/ChatBot.svelte

```diff
@@ -91,6 +95,16 @@
 			node.style.position = "relative";
 			node.appendChild(copy_div);
 		});
+
+		renderMathInElement(document.body, {
```

Comment:
I don't think we want to render Math everywhere in `document.body`, just in the Chatbot element itself, correct? (Could be misunderstanding what this element does)

Reply:
Yeah. We could limit this to the container of the chat bot. Any parent element in this file should be fine in place of `document.body`